static/scripts/onboarding/onboarding.ts

```diff
@@ -27,102 +29,21 @@ const REPO_NAME = "ubiquibot-config";
 const DEFAULT_REPO = "ubiquibot";
 const KEY_PATH = ".github/ubiquibot-config.yml";
 const DEFAULT_PATH = "ubiquibot-config-default.json";
-const KEY_NAME = "private-key-encrypted";
+const PRIVATE_ENCRYPTED_KEY_NAME = "privateKeyEncrypted";
```

Comment:
Shouldn't it be `evmPrivateEncrypted` (as stated [here](https://github.com/pavlovcik/ubiquibot/blob/e402d1652a74050d86d62dfb3b68310cbf3d2696/src/types/configuration-types.ts#L65))?

Reply:
It will be when we target the newest bot conf, right now it's using the one stored in ubiquibot not pavlovcik

Reply:
Ok, if you have a plan then it's fine.

Right now, as far as I understand, the production bot instance is served from https://github.com/pavlovcik/ubiquibot/tree/refactor/move-to-delegated-compute which uses `evmPrivateEncrypted` hence if you generate a new config via the current PR it will be invalid.

Reply:
Discussed [here](https://github.com/ubiquibot/configuration/issues/1) Hopefully will have a look at it tomorrow 😄